Library/Formula/gtimer.rb

```diff
@@ -0,0 +1,84 @@
+class Gtimer < Formula
+  desc "GTimer is a GTK application for timing how you spend your time among various projects and tasks with support for generating plain text and HTML reports."
+  homepage "http://www.k5n.us/gtimer.php"
+  url "http://downloads.sourceforge.net/project/gtimer/2.0.0/gtimer-2.0.0.tar.gz"
+  version "2.0.0"
+  sha256 "a2dd70793a5b1b4d497be0c4429b8e8cef6910497b6af8fbdedff00764308458"
+
+  depends_on :x11
+  depends_on "gtk+"
+
+
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!